sql/catalyst/src/main/java/org/apache/spark/sql/vectorized/ArrowColumnVector.java

```diff
@@ -182,6 +182,8 @@ void initAccessor(ValueVector vector) {
       accessor = new TimestampAccessor(timeStampMicroTZVector);
     } else if (vector instanceof TimeStampMicroVector timeStampMicroVector) {
       accessor = new TimestampNTZAccessor(timeStampMicroVector);
+    } else if (vector instanceof TimeNanoVector timeNanoVector) {
```

Comment:
Just a stupid question, when is these method used? why should we change arrow related method when adding connect dataType?

Reply:
No, it isn't a stupid question. Arrow is used to stream data between connect server and clients: (https://spark.apache.org/docs/latest/spark-connect-overview.html#how-spark-connect-works)